tests/run/cython3.pyx

```diff
@@ -578,15 +578,15 @@ def annotation_syntax(a: "test new test", b : "other" = 2, *args: "ARGS", **kwar
     >>> len(annotation_syntax.__annotations__)
     5
     >>> print(annotation_syntax.__annotations__['a'])
-    test new test
+    u'test new test'
```

Comment:
I don't think it does any harm. It's added by the code generation in `AnnotationWriter` (or it's base classes). The only real reason to change it would be that it generates a slightly different representation to Python, but since equivalent representations are OK in the PEP I don't think it's a big problem.

Reply:
It's not what users of Python 3.x expect, though. Cython will have to adapt to that at some point, and 3.0/1 seems a good time to do it, better now than in 3.2.